clientlib/src/main/proto/yelp/nrtsearch/luceneserver.proto

```diff
@@ -113,6 +113,14 @@ service LuceneServer {
       body: "*"
     };
 
+    }
+    /* Sync an index state */
+    rpc syncIndexState (SyncIndexStateRequest) returns (DummyResponse) {
```

Comment:
Whatever this endpoint ends up being, it would be good to have a cli command to make it easy to invoke locally.